handbook/company/pricing-features-table.yml

```diff
@@ -1133,23 +1109,10 @@
   usualDepartment: IT
   productCategories: [Device management]
   pricingTableCategories: [Device management]
-- industryName: API first by design
```

Comment:
Removed because it is not an MDM feature and is duplicative of REST API from the integrations section.